### DISCUSSION TITLE
1024x1024 icon not installed for Linux

### DISCUSSION THREAD

**Discussion by rivenirvana** [ghostty-org/ghostty#3856] [Feature Requests, Ideas] [answered]
I'm wondering if leaving it out was intentional, but I doubt it since the size is supported. But in the new icon commit (247409d70577364dca801f3512b0802b03599444), a 1024x1024 size icon was introduced for use in Linux but was left unused by `build.zig`. I'm maintaining a COPR package for this and that's when I noticed, but instead of manually installing it in the package spec, I figured it'd be better to let the upstream build file handle this as well if it was unintentionally left out.

If so, I'd like to make a PR with these changes (https://github.com/ghostty-org/ghostty/compare/main...rivenirvana:ghostty:install-icon-1024). Let me know if there's anything I missed, but I can confirm that this builds properly.

**Comment by mitchellh** [comment, marked as answer]
Approved #3967
